docs/development/compatibility-policy.rst

```diff
@@ -319,6 +319,8 @@ Classes
 
 Classes are deprecated by raising a warning when they are access from within their module, using the :py:func:`deprecatedModuleAttribute <twisted.python.deprecate.deprecatedModuleAttribute>` helper.
 
+This can trigger the warnings at important time.
```

Comment:
You mean _import_ time, right? But I don't think that this sentence is adding anything.
```suggestion
```